### ISSUE TITLE
@fluent/react v0.11.0 breaks SSR

### ISSUE THREAD

**Issue by rlr** [projectfluent/fluent.js#452]
I tried to fix this but it's more involved than I initially thought.

`ReactContext` always initializes a `ReactLocalization` with no params. This in turn triggers a `createMarkupParser()` call which requires a `document` and blows up during SSR:
https://github.com/projectfluent/fluent.js/blob/c7c39dadb8adeb47a542ddb9e3821dfec5a6f38d/fluent-react/src/context.js#L4
https://github.com/projectfluent/fluent.js/blob/c7c39dadb8adeb47a542ddb9e3821dfec5a6f38d/fluent-react/src/localization.js#L17

In summary, I can't seem to avoid this error during SSR:
https://github.com/projectfluent/fluent.js/blob/c7c39dadb8adeb47a542ddb9e3821dfec5a6f38d/fluent-react/src/markup.js#L10-L17

**Comment by rlr** [comment]
Maybe we just do:

``` Javascript
export default createContext(new ReactLocalization([], () => {}));
```
?

**Comment by stasm** [comment]
Ah, sorry about this, and thanks for filing the issue. The whole idea of introducing `createMarkupParser` was to prevent this sort of error. I'm on it.

**Comment by stasm** [comment]
I published `@fluent/react` 0.11.1 with the fix.
